src/dbmanager.cpp

```diff
@@ -57,6 +57,33 @@ bool DBManager::isNoteExist(NoteData* note)
     return query.value(0).toInt() == 1;
 }
 
+NoteData* DBManager::getNote(QString id) {
+    QSqlQuery query;
+
+    int parsedId = id.split('_')[1].toInt();
+    QString queryStr = QStringLiteral("SELECT * FROM active_notes WHERE id = %1 LIMIT 1").arg(parsedId);
+    query.exec(queryStr);
+
+    if (query.first()) {
+        NoteData* note = new NoteData();
```

Comment:
I was receiving an error for each note saying that the parent could not be set from a child thread. Maybe we could make this private to enforce internal use only?